src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -470,26 +396,29 @@ class boss_twilight_halion : public CreatureScript
                 // Using AddAura because no spell cast packet in sniffs.
                 halion->AddAura(SPELL_COPY_DAMAGE, me); // We use explicit targeting here to avoid conditions + SPELL_ATTR6_CANT_TARGET_SELF.
                 me->AddAura(SPELL_COPY_DAMAGE, halion);
-                me->AddAura(SPELL_DUSK_SHROUD, me);
+                DoCast(me, SPELL_DUSK_SHROUD, true);
```

Comment:
these two are hidden, you see the copy damage one in spell_non_melee_damage_log and tw_precision in aura updates (at least in 335)